tests/Microsoft.Bot.Streaming.Tests/Microsoft.Bot.Streaming.Tests.csproj

```diff
@@ -9,13 +9,20 @@
     <Compile Remove="ReceiveBaseTests.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
+    <FrameworkReference  Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' != 'netcoreapp3.0'">
+    <PackageReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Should this be different from the one above?

Reply:
Yes, for net core 3.0 you need to add a `FrameworkReference`, for 2.x you need a `PackageReference`. Net Core is no longer a nuget package in 3.0, is a framework, some more details [here](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/target-aspnetcore?view=aspnetcore-3.0&tabs=visual-studio#use-the-aspnet-core-shared-framework)